src/scribe_data/extract_transform/languages/German/nouns/format_nouns.py

```diff
@@ -9,6 +9,11 @@
 import json
 import os
 import sys
+PATH_TO_SCRIBE_ORG = os.path.dirname(sys.path[0]).split("Scribe-Data")[0]
+PATH_TO_SCRIBE_DATA_SRC = f"{PATH_TO_SCRIBE_ORG}Scribe-Data/src"
+sys.path.insert(0, PATH_TO_SCRIBE_DATA_SRC)
```

Comment:
Sorry - I guess I'm a little confused as to why the above is needed. Does something not work without it?

Reply:
I am unable to import the `map_genders` function in the `utils.py` file without these lines. It gives an error message `(ModuleNotFound: no module named scribe_data)`

Reply:
I'll work to figure this out after the merge, @ikeadeoyin :)